File: mooc-2024/part-4/4-define-iteration/06-the-sum-of-positive-numbers/main.py
def sum_of_positives(l):
    s = 0
    for el in l:
        if el > 0:
            s = s + el
    return s

my_list = [1, -2, 3, -4, 5]
result = sum_of_positives(my_list)
print("The result is", result)

File: mooc-2024/part-4/4-define-iteration/06-the-sum-of-positive-numbers/main.c
#include <stdio.h>

int main()
{
    int sum = 0;
    int len = 5;
    int arr[] = {1, -2, 3, -4, 5};

    for (int idx = 0; idx < len; idx++)
    {
        if (arr[idx] > 0)
        {
            sum += arr[idx];
        }
    }

    printf("The result is %d\n", sum);

    return 0;
}